docs/source/upgrading.rst

```diff
@@ -117,6 +118,31 @@ the format as you deploy a given stage.
   ``.chalice/deployed.json`` as well.
 
 
+The ``chalice package`` command has also been updated to use the
+deployer.  This results in several changes compared to the previous
+version:
+
+* Pure lambdas are supported
+* Scheduled events are supported
+* Parity between the behavior of ``chalice deploy`` and ``chalice package``
+
+As part of this change, the CFN resource names have been updated
+to use ``CamelCase`` names.  Previously, chalice converted your
+python function names to CFN resource names by removing all
+non alphanumeric characters and appending an md5 checksum,
+e.g ``my_function -> myfunction3bfc``.  With this new packager
+update, the resource name would be converted as
+``my_function -> MyFunction``.  Note, the ``Outputs`` section
+renames unchanged in order to preserve backwards compatibility.
+In order to fix parity issues with ``chalice deploy`` and
+``chalice package``, we now explicitly create an IAM role
+resource as part of the default configuration.  This means
+the default SAM template requires capability ``CAPABILITY_NAMED_IAM``
+instead of ``CAPABILITY_IAM``.  As always, you still have the
```

Comment:
It would be good to be more explicit in what people may have to do to potentially fix this if they have the wrong capabilities set.

Reply:
You know what, I don't actually need to use `CAPABILITY_NAMED_IAM`.  It doesn't really matter if we don't use the same role name used by `chalice deploy`.  I think the important point is that the role to lambda function mapping (and their associated policies) are what matter.  Letting cloudformation autogenerate the role is actually fine.  I'm going to update the PR to account for this.